app/client/cli/account.go

```diff
@@ -2,15 +2,14 @@ package cli
 
 import (
 	"fmt"
-
+	"github.com/pokt-network/pocket/app/client/keybase/debug"
 	"github.com/pokt-network/pocket/shared/crypto"
 	"github.com/pokt-network/pocket/utility/types"
 	"github.com/spf13/cobra"
 )
 
 func init() {
 	accountCmd := NewAccountCommand()
-	accountCmd.Flags().StringVar(&pwd, "pwd", "", "passphrase used by the cmd, non empty usage bypass interactive prompt")
```

Comment:
Can you look into `cli/actor.go` and follow the same pattern w.r.t attaching the password params: 
1. Use options to set a pattern in case we need to add others
2. Don't embed it deep inside of `accountCommands` so its easier to find

```go
func init() {
	rootCmd.AddCommand(NewActorCommands(attachPwdFlagToSubcommands())...)

	rawChainCleanupRegex = regexp.MustCompile(rawChainCleanupExpr)

	oneMillion = big.NewInt(1e6)
}
```

Reply:
@Olshansk without adding excess code the `gov.go` and `account.go` files don't need I feel this is the best solution:

```go
func NewAccountCommand() *cobra.Command {
	cmd := &cobra.Command{
		Use:     "Account",
		Short:   "Account specific commands",
		Aliases: []string{"account"},
		Args:    cobra.ExactArgs(0),
	}

	cmds := accountCommands()
	applySubcommandOptions(cmds, attachPwdFlagToSubcommands())
	cmd.AddCommand(cmds...)

	return cmd
}
```

similar with the gov one.

The way the actor one works is quite different from the others as it uses the `actorCmdDef` struct which is unique to this CLI endpoint - the others don't have something like this and its definitely not needed (as the code is now) for them either so they shouldn't go in the `init()` function as it would require changes that I don't really think are of any benefit to the structure of the other endpoints. Happy to do it if you think this move is not enough though.

Reply:
Great explanation and thanks for putting so much thought into it. The current solution LGTM